pkg/capabilities/capabilities.go

```diff
@@ -187,7 +224,7 @@ func (c *Capabilities) EBPF(cb func() error) error {
 	c.lock.Lock()
 	defer c.lock.Unlock()
 
-	if !c.bypass {
+	if !c.bypass || !c.baseEbpf {
```

Comment:
Shouldn't this be `&&`?
We want to enter here only if both bypass and base ebpf is false, no?

Reply:
Yep you're right.